src/dispatch/static/dispatch/src/components/LockButton.vue

```diff
@@ -0,0 +1,79 @@
+<!-- LockButton.vue -->
+<template>
```

Comment:
This looks specific to cases, we should move it or make it more general such that we can use it for incidents as well.